### PR TITLE
Allow user custom fzf tags layout

### PR DESCRIPTION
**Pull request by luathn**
This will allow user to customize fzf_tags layout.
It won't be stick with `down: 40%`.
Please take a look.

### CODE REVIEW

File: autoload/fzf_tags.vim [hunk 1]
```diff
@@ -9,6 +9,8 @@ let s:actions = {
   \ 'ctrl-x': 'split',
   \ 'ctrl-v': 'vsplit' }
 
+let s:default_layout = { 'down': '~40%' }
```
Comment (zackhsi) [inline autoload/fzf_tags.vim:12]: Hmm, the more I think about it, the more I think we should get out of the way and let the normal FZF configuration do its thing.

The FZF experience should be consistent across all the tools that use it. How about we remove the default layout altogether, and in the README we point the user to the global FZF configurations?

https://github.com/junegunn/fzf/blob/master/README-VIM.md#configuration

Would this solve your layout customization need?

**Review by zackhsi — CHANGES_REQUESTED** [review]
Hey! I'm hoping we can leverage the existing FZF configuration points where possible and avoid introducing new ones. LMK your thoughts.

File: autoload/fzf_tags.vim (hunk 1 above)
Comment (funbringer) [reply autoload/fzf_tags.vim:12]: @zackhsi +1 for the default FZF look. I've just spent 20 minutes to find and remove this `~40%` thing (i'm just a casual fzf user, so bear with me :)

Comment (zackhsi) [reply autoload/fzf_tags.vim:12]: @funbringer I've addressed this in master by using the pop-up layout as default, and supporting the user's g:fzf_layout setting.

See:
- https://github.com/zackhsi/fzf-tags/issues/13
- https://github.com/zackhsi/fzf-tags/pull/14

**Pull request closed without merging**